src/services/types.ts

```diff
@@ -1571,6 +1573,13 @@ export interface OutliningSpan {
     kind: OutliningSpanKind;
 }
 
+/** @internal */
+export interface MapCodeDocumentMapping {
+    fileName: string;

```

Comment:
```suggestion
```
So that its inline with other service APIs

Reply:
In fact you could get rid of this and instead add these as parameters to API as well